.github/actions/setup-go/action.yml

```
@@ -1,9 +1,20 @@
 name: Setup go
 
+outputs:
+  go-version:
+    description: "The Go version that was set up"
+    value: ${{ steps.setup-go.outputs.go-version }}
+
 runs:
   using: 'composite'
   steps:
     - name: Setup go
+      id: setup-go
       uses: actions/setup-go@v4
```

As we are on it, we can probably bump this version as well

```suggestion
      uses: actions/setup-go@v5
```